src/lib/components/layouts/Project.svelte

```diff
@@ -23,6 +23,7 @@
   export let addons: Promise<APIResponse<Page<AddOnListItem>>>;
 
   $: projectQuery = `+project:${project.id}`;
+  $: combinedQuery = `${projectQuery} ${query}`.trim();
```

Comment:
Since `projectQuery` is only used by `combinedQuery`:

```suggestion
  $: combinedQuery = `+project:${project.id} ${query}`.trim();
```